### PR TITLE
Create LICENSE-File

### PR DESCRIPTION
**Pull request by cmllr**
You need this file that GitHub can detect your license in the project info!

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2016 Marvin Menzerath
```
Comment (menzerath) [inline LICENSE]: Would you kindly change `2016` to `2014-2016` to reflect the long history of this project?

**Review by menzerath — CHANGES_REQUESTED** [review]
Just one small change...

**Comment by cmllr** [comment]
Done!

**Review by menzerath — APPROVED** [review]
Thanks!

**Pull request merged**